scss/bootstrap/components/_checkbox-switch.scss

```diff
@@ -0,0 +1,104 @@
+.checkbox-switch {
+  padding-top: 0 !important;
+  position: relative;
+
+  input {
+    bottom: 0;
+    height: 0;
+    left: 0;
+    margin: 0;
+    right: 0;
+    top: 0;
+    visibility: hidden;
+    width: 0;
+
+    + span {
+      cursor: pointer;
+      padding-left: $crm-checkbox-switch-size-normal * 2;
+      user-select: none;
+    }
+
+    + span::before {
+      background-color: $crm-gray-matte;
+      border-radius: $crm-checkbox-switch-size-normal;
+      content: '';
+      display: inline-block;
+      height: $crm-checkbox-switch-size-normal;
+      left: 0;
+      position: absolute;
+      transition: background-color $crm-checkbox-switch-speed ease-out;
+      width: $crm-checkbox-switch-size-normal * 2;
+    }
+
+    + span::after {
```

Comment:
```scss
+ span {
  &::before {

  }

  &::after {
  }
}
```
also, it seems there is some common styling between `::before` and `::after`

```scss
+ span {
  &::after,
  &::before {
    // common style
  }

  &::before {

  }

  &::after {
  }
}
```

Reply:
Done